tests/tools/template_testvectors.json

```diff
@@ -50,22 +50,25 @@
         "kind": "RegisterEntity",
         "signature_context": "oasis-core/consensus: tx for chain test-chain-id",
         "tx": {
+            "fee": {
```

Comment:
Not sure we need to add fee here. If this is necessary for the test to work, this is indicating some kind of bug.

Reply:
No it wasn't needed. No sure why I have added it but without fee test is passing.